### PR TITLE
[DOC-396] Set up SSO using Jumpcloud

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-cloud/managed-security/managed-authentication/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 1]
```diff
@@ -178,6 +178,59 @@ To configure federated authentication in YugabyteDB Managed, do the following:
 1. Enter the Okta domain for your application.
 1. Click **Enable**.
 
+You are redirected to sign in to your IdP to test the connection. After the test connection is successful, federated authentication is enabled.
+
+  {{% /tab %}}
+
+  {{% tab header="Jumpcloud" lang="Jumpcloud" %}}
+
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:186]: There is a list of supported IdPs at start of section, make bulleted and add JC

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 2]
```diff
@@ -178,6 +178,59 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+To use Jumpcloud for your IdP, do the following:
+
+1. Sign in to your Jumpcloud account and create an application.
+
+    - Log in to Jumpcloud with an administrator account.
+    - Under **SSO Applications**, click **Add New Application**.
+    - Select **Custom Application**, and make sure the integration supports "SSO with OIDC" on the next page.
+    - Under **Manage Single Sign-On (SSO)**, select **Configure SSO with OIDC**, and click **Next**.
+    - Under **Enter General Info**, add the application name (for **Display Label**), **Description** and logo (for **User Portal Image**), and select **Show this application in User Portal**; the information gets displayed as a tile when you log in the next time.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:197]:
```suggestion
    - Under **Enter General Info**, add the application name (for **Display Label**), **Description**, and logo (for **User Portal Image**), and select **Show this application in User Portal**.
    
    This information is displayed as a tile when users sign in to YugabyteDB Aeon.
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 3]
```diff
@@ -178,6 +178,59 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+
+  {{% tab header="Jumpcloud" lang="Jumpcloud" %}}
+
+**Create an application in Jumpcloud**
+
+To use Jumpcloud for your IdP, do the following:
+
+1. Sign in to your Jumpcloud account and create an application.
+
+    - Log in to Jumpcloud with an administrator account.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:193]:
```suggestion
    - Sign in using an administrator account.
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 4]
```diff
@@ -178,6 +178,59 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+
+    Under **Attribute Mapping**, for **Standard Scopes**, select **Email** and **Profile**.
+
+    Click **Activate** when you are done. You will get a pop up to save the **Client ID** and **Client Secret**.
+
+1. Configure Attributes and Identity Management as required.
+
+1. Integrate the user in Jumpcloud.
+
+    - Navigate to **User Groups**, select the user groups you want to access YugabyteDB Managed, and click **Save** when you are done.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:216]:
```suggestion
    - Navigate to **User Groups**, select the user groups you want to access YugabyteDB Aeon, and click **Save** when you are done.
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 5]
```diff
@@ -178,6 +178,59 @@ To configure federated authentication in YugabyteDB Managed, do the following:
 1. Enter the Okta domain for your application.
 1. Click **Enable**.
 
+You are redirected to sign in to your IdP to test the connection. After the test connection is successful, federated authentication is enabled.
+
+  {{% /tab %}}
+
+  {{% tab header="Jumpcloud" lang="Jumpcloud" %}}
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:185]:
```suggestion
  {{% tab header="JumpCloud" lang="jumpcloud" %}}
```

Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:185]: JumpCloud everywhere

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 6]
```diff
@@ -178,6 +178,59 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+
+    Under **Endpoint Configuration**, do the following:
+
+    - **Redirect URIs** - enter `https://yugabyte-cloud.okta.com/oauth2/v1/authorize/callback`.
+    - **Client Authentication Type** - select **Client Secret Post**.
+    - **Login URL** - enter `https://cloud.yugabyte.com/login`.
+
+    Under **Attribute Mapping**, for **Standard Scopes**, select **Email** and **Profile**.
+
+    Click **Activate** when you are done. You will get a pop up to save the **Client ID** and **Client Secret**.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:210]:
```suggestion
    Click **Activate** when you are done.
    
    You will be prompted in a pop up to save the **Client ID** and **Client Secret**. Save these in a secure location, you will need to provide these credentials in YugabyteDB Aeon.
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 7]
```diff
@@ -178,6 +178,59 @@ To configure federated authentication in YugabyteDB Managed, do the following:
...
+
+1. Configure Attributes and Identity Management as required.
+
+1. Integrate the user in Jumpcloud.
+
+    - Navigate to **User Groups**, select the user groups you want to access YugabyteDB Managed, and click **Save** when you are done.
+
+To configure Jumpcloud federated authentication in YugabyteDB Managed, you need the following application properties:
+
+- Client ID and secret of the application you created. Note that you will get a pop up to save the **Client ID** and **Client Secret** when you activate your application. Alternatively, the **Client ID** is provided on the **SSO** tab.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:220]:
```suggestion
- Client ID and secret of the application you created. Note that you will get a pop up to save the **Client ID** and **Client Secret** when you activate your application. The **Client ID** is also displayed on the **SSO** tab.
- The Authorization URL for your application. [where is this displayed?]
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 8]
```diff
@@ -178,6 +183,64 @@ To configure federated authentication in YugabyteDB Aeon, do the following:
...
+1. Configure Attributes and Identity Management as required.
+
+1. Integrate the user in JumpCloud.
+
+    - Navigate to **User Groups**, select the user groups you want to access YugabyteDB Aeon, and click **Save** when you are done.
+
+To configure JumpCloud federated authentication in YugabyteDB Managed, you need the following application properties:
+
+- Client ID and secret of the application you created. Note that you will get a pop up to save the **Client ID** and **Client Secret** when you activate your application. The **Client ID** is also displayed on the **SSO** tab.
+- The Authorization URL for your application. [where is this displayed?]
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:230]: Need to add where they can find this - where does the Authorization URL come from/where do they find it?
Is it the Redirect URI they entered above, or something else?

Comment (ddhodge) [reply docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:230]: @Aastha-yb

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 9]
```diff
@@ -178,6 +183,64 @@ To configure federated authentication in YugabyteDB Aeon, do the following:
...
+1. Sign in to your JumpCloud account and create an application.
+
+    - Sign in using an administrator account.
+    - Under **SSO Applications**, click **Add New Application**.
+    - Select **Custom Application**, and make sure the integration supports "SSO with OIDC" on the next page.
+    - Under **Manage Single Sign-On (SSO)**, select **Configure SSO with OIDC**, and click **Next**.
+    - Under **Enter General Info**, add the application name (for **Display Label**), **Description**, and logo (for **User Portal Image**), and select **Show this application in User Portal**.
+
+    This information is displayed as a tile when users sign in to YugabyteDB Aeon.
+    - Click **Save Application**.
```
Comment (atharva-yb) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:205]:
```suggestion
    - Click **Configure Application**.
```

https://github.com/yugabyte/yugabyte-db/assets/140231703/9ceefd9c-0e0c-4f49-b3bd-068fcc4a32f2

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md [hunk 10]
```diff
@@ -178,6 +183,64 @@ To configure federated authentication in YugabyteDB Aeon, do the following:
...
+
+1. Configure your application.
+
+    Under **Endpoint Configuration**, do the following:
```
Comment (atharva-yb) [inline docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:209]:
```suggestion
    Under **SSO > Endpoint Configuration**, do the following:
```

File: docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md (hunk 8 above)
Comment (atharva-yb) [reply docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:230]: Authorization URL is not required. We should remove this point.

Comment (Aastha-yb) [reply docs/content/preview/yugabyte-cloud/managed-security/managed-authentication.md:230]: No Authorization URL needed. This was initially discussed to be added and exists in the POC doc, but later we decided to not keep it as it is a constant URL. Pls refer the following ss for more context on the info needed at YBM side :

**Review by Aastha-yb — APPROVED** [review]

**Pull request merged**